scripts/extensions/annotationsLibrary/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "main": "dist/extension.js",
+  "scripts": {
+    "compile": "tsc",
+    "watch": "tsc -watch"
+  },
```

Comment:
How does it work when you're developing an extension? Does `grunt` watch and compile extensions automatically?

Reply:
Grunt only watches the extension output and reloads when it changes, but doesn't know about the link between extension code and output. You have to run both grunt and `npm run watch` inside the extension folder. Extensions have a standalone build process by TypeScript and do not depend on core for anything but extracting translations from `gettext` calls.

Reply:
okay, good to know 👍